IntelFsp2Pkg/FspSecCore/Ia32/FspApiEntryT.nasm

```diff
@@ -21,7 +21,7 @@ extern   ASM_PFX(PcdGet32 (PcdFspReservedBufferSize))
 ; Following functions will be provided in PlatformSecLib

```

Comment:
@srisrid changes look fine. But for tracking purposes, can you explain why you are cherry picking these into 202208?